clickhouse/types/types.h

```diff
@@ -34,6 +34,7 @@ class Type {
         FixedString,
         DateTime,
         Date,
+        Date32,
```

Comment:
Please move the new value to the end. This is `enum` is visible to users and some may rely on order of elements. I think we can re-arrange those to look better in a next major version, like `3.0`

Reply:
Done